app/controllers/hub/assigned_clients_controller.rb

```diff
@@ -10,7 +10,7 @@ class AssignedClientsController < ApplicationController
 
     def index
       @page_title = I18n.t("hub.assigned_clients.index.title")
-      @clients = filtered_and_sorted_clients
+      @clients = filtered_and_sorted_clients.with_eager_loaded_associations
```

Comment:
Does it make sense to call `.with_eager_loaded_associations` in `filtered_and_sorted_clients`?